.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 .ipynb_checkpoints
 *.tsbuildinfo
 *.lock
+**/node_modules/**/README.md
```

Comment:
Looks like this should be two separate lines.

Reply:
embedme was picking node_modules/rimraf/README.md and printing warning, asking to add in .gitignore if we wanted to skip that. I will add a comment to this line to make it more clear.

Reply:
Ah right indeed, then we should keep it like this :+1: